csharp/ql/lib/semmle/code/dotnet/Declaration.qll

```diff
@@ -4,11 +4,15 @@
 
 import Element
 import Type
+private import semmle.code.csharp.commons.QualifiedName
 
 /** A declaration. */
 class Declaration extends NamedElement, @dotnet_declaration {
-  override predicate hasQualifiedName(string qualifier, string name) {
-    qualifier = this.getDeclaringType().getQualifiedName() and
+  override predicate hasQualifiedName(string namespace, string name) {
```

Comment:
OK, I see now that renaming the parameter from `qualifier` to `namespace` is a bad idea, since it is not actually a namespace here (it is a namespace concatenated with a type).

Reply:
Should I change it back to `qualifier` for hasQualifiedName/2?

Reply:
Yes, I think so.